server/controllers/user.js

```diff
@@ -21,3 +21,29 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ users: users });
 });
+
+// @route GET /users/isSitter/
+// @desc GET allowed parameters within User model
+// @access Private
+exports.getIsSitter = asyncHandler(async (req, res, next) => {
+  const id = req.user.id;
+
+  const user = await User.findById(id);
+  const { isSitter } = user;
+
+  res.status(200).json({ isSitter });
+});
```

Comment:
Alternatively, you could have modified the token(s) in `auth` controller (as well as `useAuthContext` on frontend) to include `isSitter` field as well.

Reply:
Thank you so much! I was trying to figure out how to do that!